### ISSUE TITLE
Firefox Rendering Issues

### ISSUE THREAD

**Issue by ghost** [Code4HR/open-health-inspection-app#204]
As of this posting, OHI next doesn't render appropriately in Firefox 35.0a2 (2014-10-21) on Windows 7 64-bit.
![ohinext](https://cloud.githubusercontent.com/assets/5629268/4756103/ed180cf6-5aca-11e4-827c-187f97214f01.png)

I have disabled Google Analytics and allowed everything else.

**Comment by ghost** [comment]
Not sure if this helps, but I tested this from my work computer which has an out-of-state network, and I did in my console log on Firefox get the following error logs:

```
"DETACHING CONTENT SCRIPT" "http://next.openhealthinspection.com/#/" content-script.js:30
"getting location" production.js:95
"value=" undefined production.js:418
TypeError: setting a property that has only a getter production.js:135
"Coming from out of state or geolocation unavailable." production.js:132
```

Maybe this issue stems from out-of-state visitors?

**EDIT**: Got the same error messages for the current live deployment as well.

**Comment by wbprice** [comment]
I can't replicate the above in Firefox 32.x (OSX).  I'll try again when I get home.  Aside from that, it looks like the fallbacks for geolocation failure were broken at some point in the past.

**Comment by ghost** [comment]
Possibly.  Not sure how much OS X Firefox differs from the Windows version, but worth a shot.

**Comment by wbprice** [comment]
As we discovered in #209/#210, geolocation failure wasn't being handled correctly.  The scrollbars are a consequence of using 'overflow: auto;' where we ought to be using 'overflow: visible'.

Reference:
https://developer.mozilla.org/en-US/docs/Web/CSS/overflow

**Comment by wbprice** [comment]
Fixed in #215.
